### PR TITLE
api: deprecate s3 operator flag

### PR DESCRIPTION
**Pull request by fanminshi**
Ref #1377 
[skip ci]

### CODE REVIEW

**Comment by hongchaodeng** [comment]
Also add to changelog

**Comment by fanminshi** [comment]
@xiang90 @hongchaodeng I added a note mentioning that operator lvl support will be drop in v0.5.4.

**Comment by fanminshi** [comment]
@hongchaodeng added this to changelog.

File: doc/user/backup_config.md [hunk 1]
```diff
@@ -49,7 +49,9 @@ Saving backups to S3 is also supported. The S3 backup policy can be set at two l
 
 If configurations for both levels are specified then the cluster level configuration will override the operator level configuration.
 
-### Operator level configuration  
+### DEPRECATED - Operator level configuration 
+
+**Note: Operator level configuration will be completely removed in the future release v0.5.4; use [Cluster level configuration](#cluster-level-configuration) instead.**
```
Comment (hongchaodeng) [inline doc/user/backup_config.md]: > future release v0.5.4

v0.6.0

**Comment by hongchaodeng** [comment]
lgtm after nit

**Pull request merged**